include/libsemigroups/containers.hpp

```diff
@@ -621,6 +621,131 @@ namespace libsemigroups {
       }
     };
 
+    // Currently only supports N x N 2-dimensional static vectors
```

Comment:
Delete this comment, it clearly applies to something else.

Reply:
Maybe add a comment that `StaticVector1` has the same interface as `std::vector` (some of it at least) but is really just an array.